### PR TITLE
Skip integration tests with the flag `--no-integration-tests`

### PR DESCRIPTION
**Pull request by marcofavorito**
## Proposed changes

Calling `pytest` with the flag `--no-integration-tests` all the tests that require an OEF node will be skipped.

The same option, in the future, can be used to skip tests that requires specific fixtures.

## Fixes

Partially addresses issue #63 

## Types of changes

What types of changes does your code introduce to agents-tac?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [X] I have read the [CONTRIBUTING](../master/CONTRIBUTING.rst) doc
- [X] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [X] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

None.

### CODE REVIEW

File: tests/test_agent.py [hunk 1]
```diff
@@ -1,87 +0,0 @@
...
-#   Unless required by applicable law or agreed to in writing, software
-#   distributed under the License is distributed on an "AS IS" BASIS,
-#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#   See the License for the specific language governing permissions and
-#   limitations under the License.
-#
-# ------------------------------------------------------------------------------
-
-"""This test module contains the tests for the OEF communication using an OEF."""
-import time
```
Comment (DavidMinarsch) [inline tests/test_agent.py]: Why has this been removed?

Comment (marcofavorito) [reply tests/test_agent.py]: because similar tests are in `tests/test_channel/test_oef/test_communication.py`.

Comment (marcofavorito) [reply tests/test_agent.py]: in particular, in the class `TestDefault`.

Comment (DavidMinarsch) [reply tests/test_agent.py]: ok didn't see that.

**Pull request merged**